### PR TITLE
Deprecate /masternode/list -/validate endpoints

### PR DESCRIPTION
**Pull request by Cofresi**
responding with a message explaining the deprecation.

We'll add the new endpoints in a later PR

### CODE REVIEW

**Review by shumkov — CHANGES_REQUESTED** [review]
Please do not comment code. Just remove it. If we need it we always can use git history.

**Comment by Cofresi** [comment]
> Please do not comment code. Just remove it. If we need it we always can use git history.

done!

File: package.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
   "name": "@dashevo/insight-api",
   "description": "A Dash blockchain REST and WebSocket API Service",
-  "version": "2.2.9",
+  "version": "2.3.0",
```
Comment (shumkov) [inline package.json:4]: Don't we need to increase major version if we remove method or it didn't work anyway so we kinda keep backward compatibility? :)))

Comment (Cofresi) [reply package.json:4]: yeah, kind of. Honestly don't know what is best here :). I'm fine with any bump. Which do you prefer? Major, minor or patch?

Comment (shumkov) [reply package.json:4]: There are more complications. For example, you need to bump UI version too. Let’s keep it as is.

**Review by shumkov — APPROVED** [review]
👍 utACK

**Pull request merged**